prow/plugins/assign/assign.go

```diff
@@ -134,13 +157,53 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	if len(unassign) > 0 {
 		log.Printf("Removing assignees from %s/%s#%d: %v", ae.org, ae.repo, ae.number, unassign)
 		if err := gc.UnassignIssue(ae.org, ae.repo, ae.number, unassign); err != nil {
-			return err
+			msg := fmt.Sprintf("Unassign failed: %v.", err.Error())
```

Comment:
This is what https://github.com/kubernetes/test-infra/blob/master/prow/plugins/lgtm/lgtm.go#L85-L95 is doing, I just feel like in this way we can give more informative comments. Or the lower level Assign/UnassignIssue need to handle it?